### PR TITLE
Fueled/shopify fueled

### PR DESCRIPTION
**Pull request by seanberto**
### Proposed changes

Documenting Fueled's Shopify Source integration for Segment.

### Merge timing

- ASAP once approved

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Fueled's Shopify Source streams events into Segment. It uses server-side methods to send `track` , `identify` , and `page` events to Segment. These events are then available in any Segment Destination that accepts server-side events, and are available in a schema in your data warehouse, so you can query using SQL.
+
+The default behavior is for Fueled to pass the email address of identified users as the Segment `userId` , when it's available.
+
+## Events
+
+The table below lists events that Fueled's Shopify Source automatically sends to Segment. These events appear as tables in your warehouse, and as regular events in other Destinations. Fueled's Shopify Source includes the `userId` (the Shoper customer's email address) if available.
+
+Fueled's Shopify Source also supports *custom events* captured client-side by our javascript library and sent to Segment. To learn more about sending custom events to Segment, visit this [Learning Center article](https://learn.fueled.io/apps/shopify/shopify-event-tracking/customizing-client-side-shopify-events){:target="_blank”}.
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:43]: who does "our" refer to?

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+### Fueled Standard Events
+
+Fueled's Shopify Source integration follows Segment's [eCommerce event specification](https://segment.com/docs/connections/spec/ecommerce/v2/){:target="_blank”}.
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:47]:
```suggestion
Fueled's Shopify Source integration follows Segment's [eCommerce event specification](/docs/connections/spec/ecommerce/v2/).
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+The default behavior is for Fueled to pass the email address of identified users as the Segment `userId` , when it's available.
+
+## Events
+
+The table below lists events that Fueled's Shopify Source automatically sends to Segment. These events appear as tables in your warehouse, and as regular events in other Destinations. Fueled's Shopify Source includes the `userId` (the Shoper customer's email address) if available.
+
+Fueled's Shopify Source also supports *custom events* captured client-side by our javascript library and sent to Segment. To learn more about sending custom events to Segment, visit this [Learning Center article](https://learn.fueled.io/apps/shopify/shopify-event-tracking/customizing-client-side-shopify-events){:target="_blank”}.
+
+### Fueled Standard Events
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:45]:
```suggestion
### Fueled standard events
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+| Checkout Started     | User initiated the order process                 |
+| Order Completed      | User completed the order                         |
+| Order Updated        | User updated the order                           |
+| Product Added        | User added a product to their shopping cart      |
+| Product Removed      | User removed a product from their shopping cart  |
+| Product List Viewed  | User viewed a product list or collection         |
+| Product Viewed       | User viewed a product details                    |
+| Products Searched    | User searched for products                       |
+
+## Event Properties
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:67]:
```suggestion
## Event properties
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+For a complete list of all event properties and how they map to specific Shopify Storefront APIs and objects, visit: [Fueled's Shopify-to-Segment Mapping Documentation](https://learn.fueled.io/integrations/destinations/segment.com/segment-event-specifications){:target="_blank”}.
+
+## Headless Shopify Integrations
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:71]:
```suggestion
## Headless Shopify integrations
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+Fueled can be implemented on "headless" or "composable" Shopify sites. Fueled provides out-of-the-box integrations for:
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:73]:
```suggestion
Fueled can be implemented on *headless* or *composable* Shopify sites. Fueled provides out-of-the-box integrations for:
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+* Pack Digital composable Shopify sites
+* Headless Shopify sites running on TakeShape.io
+* Composable Shopify sites running on Next Commerce
+
+Contact [Fueled](mailto:support@fueled.io) for more information on these composable integrations.
+
+## Additional Shopify App Source Integrations for Segment
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:81]:
```suggestion
## Additional Shopify App source integrations for Segment
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+Fueled offers Segment Source integrations for a number of Shopify Marketplace apps. These integrations are available by contacting Fueled's customer support team at: [support@fueled.io](mailto:support@fueled.io).
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:83]:
```suggestion
Fueled offers Segment source integrations for a number of Shopify Marketplace apps. These integrations are available by contacting Fueled's customer support team at: [support@fueled.io](mailto:support@fueled.io).
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+Additional Segment integrations powered by Fueled include:
+
+* ReCharge Payments subscription events (`subscription_created`, `subscription_paused`, `subscription_canceled`)
+* Yotpo Reviews (`product_review_created`, `product_review_updated`, `site_review_created`, `site_review_updated`)
+* Gorgias customer support events (`ticket_created`, `ticket_updated`)
+* Loop Returns webhook events
+
+## Adding Destinations
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:92]:
```suggestion
## Add destinations
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+Now that your Shopify Source is set up, you can connect it with Destinations.
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:94]:
```suggestion
Now that your Shopify Source is set up, you can connect it with destinations.
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+
+Log into your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don’t appear, check the [Event Delivery](/docs/connections/event-delivery/) tool, and refer to the Destination docs for each tool for troubleshooting.
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:96]:
```suggestion
Log in to your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don’t appear, check the [Event Delivery](/docs/connections/event-delivery/) tool, and refer to the destination docs for each tool for troubleshooting.
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+
+If there are any issues with how the events are arriving to Segment, [contact the Fueled support team](mailto:support@fueled.io).
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:98]:
```suggestion
If there are any issues with how the events arrive to Segment, [contact the Fueled support team](mailto:support@fueled.io).
```

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
+title: Shopify - Powered by Fueled
+---
+
+[Fueled](https://fueled.io){:target="_blank"} is a 1st-party event collector, designed specifically for eCommerce. Fueled captures the first-party and zero-party events that fire on a Shopify website, and sends them to various destinations - most notably, Segment. Fueled leverages a combination of "client-side" and "server-side" event collection technologies to provide reliable, accurate event tracking. In addition to tracking website events in the browser, Fueled tracks "offline" purchase events, such as point-of-sale orders and subscription rebilling events triggered by Shopify apps like ReCharge Payments, Smartrr, Bold Subscritions, and Skio.
+
+With Fueled, Shopify merchants can implement Segment event tracking in minutes — without writing any code.
+
+**Shopify merchants can try our Segment integration for free for 15 days.**
```
[the remark this answers is not in the document]
Comment (seanberto) [reply src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:8]: @stayseesong - I understand that you want to remove the word "our", but your suggestion is missing an article and isn't grammatically correct. Could we say:

**Shopify merchants can try Fueled's Segment integration for free for 15 days.**

**Comment by seanberto** [comment]
@stayseesong - Thank you for the review. There's one open issue, but I've committed all of the other suggestions. Let me know next steps. Really appreciate how prompt this process has been!! This is great!!!

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
+title: Shopify - Powered by Fueled
+---
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:2]:
```suggestion
---
title: Shopify - Powered by Fueled
id: 57hcOLuW6Q
beta: true
---
```

Can you add this frontmatter please?

**Review by tcgilbert — CHANGES_REQUESTED** [review]
Just add the frontmatter, and address stacy's comments, and we should be all set!

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (seanberto) [reply src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:11]: @tcgilbert - How do we resolve @stayseesong's suggestion above? I understand the request, but the grammar in the suggestion isn't correct.

[the remark this answers is not in the document]
Comment (tcgilbert) [reply src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:11]: @seanberto your copy looks good to me! Removing one "for"

> Shopify merchants can try Fueled's Segment integration free for 15 days.

[the remark this answers is not in the document]
Comment (seanberto) [reply src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:11]: @tcgilbert there's still a missing "this" or "Fueled's" in front of the word Segment.

[the remark this answers is not in the document]
Comment (seanberto) [reply src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:11]: in @stayseesong's commit suggestion that is, @tcgilbert.

[the remark this answers is not in the document]
Comment (tcgilbert) [reply src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:11]: Ah, I see. I just edited the suggestion. Lmk if there is anything else!

**Comment by seanberto** [comment]
Updated. Hopfully ready to go!! Thanks, team!!

**Comment by seanberto** [comment]
Actually, @tcgilbert - Not sure why the PR is listed as still blocked b/c of pending change requests. Mind taking a look?

**Review by seanberto — COMMENTED** [review]
Looks good!

**Comment by seanberto** [comment]
@tcgilbert - I think I've not resolved those issues, but the merge is still blocked.

**Comment by seanberto** [comment]
I'm happy with all of your edits, @tcgilbert .

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md [hunk 15]
```diff
@@ -0,0 +1,103 @@
...
+
+Fueled's Shopify source streams events into Segment. It uses server-side methods to send `track`, `identify`, and `page` events to Segment. These events are then available in any Segment destination that accepts server-side events, and are available in a schema in your data warehouse so you can query using SQL.
+
+The default behavior is for Fueled to pass the email address of identified users as the Segment `userId` , when it's available.
+
+## Events
+
+The table below lists events that Fueled's Shopify Source automatically sends to Segment. These events appear as tables in your warehouse, and as regular events in other destinations. Fueled's Shopify Source includes the `userId` (the customer's email address) if available.
+
+Fueled's Shopify Source also supports *custom events* captured client-side by our javascript library and sent to Segment. To learn more about sending custom events to Segment, visit this [Learning Center article](https://learn.fueled.io/apps/shopify/shopify-event-tracking/customizing-client-side-shopify-events){:target="_blank”}.
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:48]: Echoing what @stayseesong put: 

By "our" javascript library, are you referring to Fueled or Segment's?

**Review by tcgilbert — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md (hunk 15 above)
Comment (seanberto) [reply src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:48]: Oh! Fueled's. Let's change that, @tcgilbert.

**Comment by seanberto** [comment]
@tcgilbert - I accidentally asked @stayseesong for an additional review. Sorry. Also, not sure how to address the question about "our" above. I was referring to Fueled's library. Can you suggest a commit?

File: src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md (hunk 15 above)
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:48]:
```suggestion
Fueled's Shopify Source also supports *custom events* captured client-side by Fueled's javascript library and sent to Segment. To learn more about sending custom events to Segment, visit this [Learning Center article](https://learn.fueled.io/apps/shopify/shopify-event-tracking/customizing-client-side-shopify-events){:target="_blank”}.
```

Comment (stayseesong) [reply src/connections/sources/catalog/cloud-apps/shopify-fueled/index.md:48]: @seanberto can you just commit this one change, and then it should be ready to go

**Comment by seanberto** [comment]
@tcgilbert and @stayseesong - I believe I have reviewed all change reqeusts, but the merging is still blocked. Not sure what I need to do.

**Review by seanberto — COMMENTED** [review]
Looks good.

**Comment by stayseesong** [comment]
I will be able to merge it after I approve. All of our PRs need approval from the docs team in order to merge, and the docs team is in charge of merging all requests

**Review by stayseesong — APPROVED** [review]

**Pull request merged**